README.md

```diff
@@ -37,9 +37,8 @@ is to add required ACCEPT rules for the target website to work properly.
 
 Contrary to most Web Application Firewalls, Naxsi doesn't rely on a
 signature base like an antivirus, and thus cannot be circumvented by an
-"unknown" attack pattern. Another main difference between Naxsi and other
-WAFs, Naxsi filters only GET and POST requests,
```

Comment:
Only `GET`, `POST` and `PATCH`. We're not handling `HEAD`, `DELETE`, …
Shouldn't we keep this part?